src/sentry/static/sentry/app/views/auth/registerForm.tsx

```diff
@@ -106,15 +106,13 @@ class RegisterForm extends React.Component<Props, State> {
             <TextField
               name="name"
               placeholder={t('Jane Bloggs')}
-              maxlength={30}
```

Comment:
This prop was never even passed to the inputs.

Reply:
Maybe worth updating it later to work, but for now, this form isn't even used.

Reply:
Could it be used by the incomplete react-only mode?

Reply:
it's not used even by that yet afaik. It will be in the future :)